src/canvas.class.ts

```diff
@@ -501,20 +501,16 @@ export class Canvas<
    */
   constructor(el: string | HTMLCanvasElement, options = {}) {
     super(el, options);
+    // @ts-ignore
+    this._initEventListeners();
+    this._createCacheCanvas();
```

Comment:
should this belong to `initElements`? Or does it relay on retina scaling being initialized?